### PR TITLE
New accordion feature

### PR DESCRIPTION
**Pull request by iSeremet-Reloadly**

### CODE REVIEW

**Review by salimzakari-reloadly — APPROVED** [review]

**Review by SabreFoxx-Reloadly — APPROVED** [review]

**Review by SabreFoxx-Reloadly — APPROVED** [review]

File: src/lib/components/accordion/accordion.module.ts [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { NgModule } from '@angular/core';
...
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
 import { CommonModule } from '@angular/common';
 
 
 
 @NgModule({
     imports: [
-        CommonModule
+        CommonModule,
+        BrowserAnimationsModule
```
[the remark this answers is not in the document]
Comment (farouk-reloadly) [reply src/lib/components/accordion/accordion.module.ts:13]: Yeah I think so

[the remark this answers is not in the document]
Comment (iSeremet-Reloadly) [reply src/lib/components/accordion/accordion.module.ts:13]: it's still not working for me on the documentation 
have you had issues also on the dashboard?

[the remark this answers is not in the document]
Comment (iSeremet-Reloadly) [reply src/lib/components/accordion/accordion.module.ts:13]: fixed now in develop, I had to remove the **BrowserAnimationsModule** also
let me know if you will have any issues in the dashboard.
version now is 41

[the remark this answers is not in the document]
Comment (salimzakari-reloadly) [reply src/lib/components/accordion/accordion.module.ts:13]: Okay thanks

**Pull request merged**